repos/tap/theme/components/definitions/definitionList.js

```diff
@@ -88,21 +88,25 @@ const listItem = theme => {
         main: {
```

Comment:
This comment is regarding `line 76` of this file.
* Can you do me a favor and change `line 76` from `pV: padSizeHalf,` to `pV: padSizeThird * 2,`
* This will give each row a bit more space around it, so things don't feel so cluttered.

Reply:
Done